sdk/compute/armcompute/example_virtualmachinescalesets_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.13
 // +build go1.13
```

Comment:
change to go1.16 to align generator
```suggestion
//go:build go1.16
// +build go1.16
```